packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -10,6 +10,7 @@ import { creatMtomContentTypeAndPayload } from "../outbound/xca/mtom/builder";
 const { log } = out("Saml Client");
 
 const timeout = 120000;
+const rejectUnauthorized = Config.isDev() ? false : true;
```

Comment:
change to make developing in dev easier

Reply:
I think this is too critical to have an alternative flow. All it takes is a bug w/ `isDev` to break an important part of certificate validation.

Isn't there another option to make it `false` while testing local that doesn't rely on something like this?

Reply:
I used the new pattern you taught me with overriding the variables instead